app.js

```diff
@@ -2,8 +2,12 @@ const express = require('express')
 const app = express()
 const {getTopics} = require('./controller/topics-controller')
 const {customErrors, psqlErrors, invalidEndpoint} = require('./error-handling')
+const endpoints =  require('./endpoints.json')
 
 app.get('/api/topics', getTopics)
+app.get('/api', (req, res, next) => {
+    res.status(200).send({endpoints})
```

Comment:
The app.js should not be responsible for sending responses, and so it would be better practice to move this line of logic to a controller